landlab/components/gravel_river_transporter/gravel_river_transporter.py

```diff
@@ -0,0 +1,484 @@
+import numpy as np
+from landlab import Component, HexModelGrid
+from landlab.grid.diagonals import DiagonalsMixIn
+from scipy.sparse.linalg import spsolve
+
+_ONE_SIXTH = 1.0 / 6.0
+
+
+def make_empty_matrix_and_rhs(grid):
```

Comment:
If this function is not used by other classes, it can be defined as a static method within the class.

Reply:
It actually could conceivably be used by other components, which is why I set it up as a non-class function. That would require moving it to something like utils, which is a TODO...

Reply:
If this is going to be used by other components, then there is no need to change them.